scheduler/src/cook/kubernetes/api.clj

```diff
@@ -176,6 +176,7 @@
     (.addContainersItem pod-spec container)
     (.setNodeName pod-spec hostname)
     (.setRestartPolicy pod-spec "Never")
+    (.setTerminationGracePeriodSeconds pod-spec 1)
```

Comment:
Why is this set, and why is it set so low? (This also feels like maybe it should be a config var)

Reply:
I just added this to speed up the teardown of the pods in k8s for testing. I'll put a TODO on it to revisit later, but for now I think it only helps us.